frontend/Dockerfile

```diff
@@ -9,6 +9,13 @@ COPY . .
 
 WORKDIR ./frontend
 
+# Workaround for ppc64le since phantomjs does not support ppc64le
+RUN if [ "$(uname -m)" = "ppc64le" ]; then \
```

Comment:
Can you explain this a little more? Why is phantomjs needed in the Docker image?

Reply:
phantomjs is a build time required package, the final docker image will not contain it.

as you can see, this is a multi-stage dockerfile, it is only used in builder not the final image.

Reply:
Right, but I'm trying to see why phantomjs is needed in the first place.
It seems like this is because we list backstopjs in our package.json? This is a dev dependency though, and isn't needed for the production build. Does the build fail on ppc64le without this fix?

Reply:
Yes, phantomjs does not support ppc64le, so I used an ibmsoe fork of phantomjs which supports ppc64le.

Reply:
I don't know if the main repo needs to support building on PowerPc though, seems like a fringe use case to me. Is this needed for your local dev workflow? Why not modify the code locally? This code is mainly used by our CloudBuild process, which runs on a regular x86 architecture.

Reply:
Ok, although we can't make sure nothing else breaks on PowerPC.